test/common.py

```diff
@@ -388,6 +388,72 @@ def metafunc(self, wasmfs, *args, **kwargs):
   return metafunc
 
 
+def also_with_nodefs(func):
+  @wraps(func)
+  def metafunc(self, fs, *args, **kwargs):
+    if DEBUG:
+      print('parameterize:fs=%s' % (fs))
+    if fs == 'nodefs':
+      self.setup_nodefs_test()
+    else:
+      self.emcc_args += ['-DMEMFS']
```

Comment:
If this one is going to define `-DMEMFS` for consistency `also_with_noderawfs()` should probably do the same?

Reply:
Done